index.d.ts

```diff
@@ -203,6 +203,10 @@ declare module "react-native-sodium" {
    * The key pair can be deterministically derived from a single key seed.
    */
   export function crypto_scalarmult_base(secretKey: string): Promise<string>;
+  export function crypto_scalarmult(
+    publicKey: string,
```

Comment:
In relation to previous comment, I think the first argument should be 'n', which contains the private key (as in the libsodium documentation, see https://doc.libsodium.org/advanced/scalar_multiplication), and the second one is 'q'.